src/main/kotlin/org/jitsi/nlj/transform/node/outgoing/TccSeqNumTagger.kt

```diff
@@ -47,7 +50,11 @@ class TccSeqNumTagger(
                         ?: rtpPacket.addHeaderExtension(tccExtId, TccHeaderExtension.DATA_SIZE_BYTES)
 
                     TccHeaderExtension.setSequenceNumber(ext, currTccSeqNum)
-                    transportCcEngine?.mediaPacketSent(currTccSeqNum, rtpPacket.length.bytes)
+
+                    val curSeq = currTccSeqNum
+                    val len = rtpPacket.length.bytes
+                    packetInfo.onSent { weakTcc.get()?.mediaPacketSent(curSeq, len) }
```

Comment:
Here we have the opportunity to get the length from the packet at send time (post-SRTP). Should we do that?

Reply:
Hm, probably not?

We compare the calculated rates that the BWE gives us with the decrypted bitrates of the received streams, so if we changed where we pass lengths to BWE we'd need also to change where we calculate the stream bitrates -- probably not worth the trouble.